### DISCUSSION TITLE
Export project,task,jobs details from the server to pdf or CSV format

### DISCUSSION THREAD

**Discussion by keshva23** [cvat-ai/cvat#4126] [Ideas]
Can we know how the data of Projects, tasks and jobs can be exported/downloaded from the database in CVAT ?

**Comment by nmanovic** [comment]
@keshva23 ,

Please look at these instructions:
https://openvinotoolkit.github.io/cvat/docs/administration/advanced/backup_guide/

**Reply by keshva23** [reply]
@nmanovic  Thank you for the reply.  

I think the requirement i mentioned is correct.

Let me correct it.

Suppose if we want to create a report of annotated Projects, tasks and jobs with time and date in pdf or CSV format.
